packages/frontend/lib/index.ts

```diff
@@ -491,7 +491,14 @@ interface ConnectionConfig {
     hmac?: string;
     user_scope?: string[];
     authorization_params?: Record<string, string | undefined>;
-    credentials?: OAuthCredentialsOverride | BasicApiCredentials | ApiKeyCredentials | AppStoreCredentials | TBACredentials | TableauCredentials;
+    credentials?:
+        | OAuthCredentialsOverride
+        | BasicApiCredentials
+        | ApiKeyCredentials
+        | AppStoreCredentials
+        | TBACredentials
+        | TableauCredentials
+        | OAuth2ClientCredentials;
```

Comment:
weird that it was missing this whole time

Reply:
@bodinsamuel , yes it is very weird 😟.